model/parVarSheet.cc

```diff
@@ -52,7 +52,9 @@ namespace minsky
             double w=0,h=0,w_prev, h_prev,lh,lw; 
             for (auto& it: itemVector)
               {
-                auto value=it->variableCast()->vValue();
+                auto v=it->variableCast();
+                auto value=v->vValue();
```

Comment:
If it is not a variable, then you have introduced a null dereference. It it  is guaranteed to be a variable, then the original code was fine. (maybe add an assertion to that fact)

Reply:
The only reason I added this is because I use the methods I added to the the VariableBase class further down in this code. But, maybe I can move those methods to the Hypercube class and then I won't have to do the above...

Reply:
OK - Probably  missed that...